task-sdk/src/airflow/sdk/api/client.py

```diff
@@ -172,10 +172,6 @@ def succeed(self, id: uuid.UUID, when: datetime, task_outlets, outlet_events, re
         )
         self.client.patch(f"task-instances/{id}/state", content=body.model_dump_json())
 
-    def heartbeat(self, id: uuid.UUID, pid: int):
-        body = TIHeartbeatInfo(pid=pid, hostname=get_hostname())
-        self.client.put(f"task-instances/{id}/heartbeat", content=body.model_dump_json())
```

Comment:
What is this move for?

Reply:
Group similar api calls. `/heartbeat` was in the middle of these `/state` calls